cmd/aasp/main.go

```diff
@@ -425,16 +425,20 @@ func main() {
 		}
 	}
 
-	// pass in EncodedUvmInformation because ciruclar reference is created if we have the following func to retrieve THIM Cert
-	// Don't look at err here.  It will always be true because not all information is encoded in kata on AKS
 	EncodedUvmInformation, _ = common.GetUvmInformation()
 
-	thimCerts, err := azure_info.CertFetcher.GetThimCerts(azure_info.CertFetcher.Endpoint)
-	if err != nil {
-		logrus.Fatalf("Failed to retrieve thim certs: %s", err.Error())
+	if common.ThimCertsAbsent(&EncodedUvmInformation.InitialCerts) {
+		logrus.Info("ThimCerts is absent, retrieving THIMCerts.")
+		thimCerts, err := azure_info.CertFetcher.GetThimCerts(azure_info.CertFetcher.Endpoint)
+		if err != nil {
+			logrus.Fatalf("Failed to retrieve thim certs: %s", err.Error())
+		}
+
+		EncodedUvmInformation.InitialCerts = *thimCerts
 	}
 
-	EncodedUvmInformation.InitialCerts = *thimCerts
+	//remove EncodedUvmReferenceInfo for now because MAA does not currently validate it
+	EncodedUvmInformation.EncodedUvmReferenceInfo = ""
```

Comment:
Let's not hard remove this.  In my commits, its possible to set the path to the referenceinfo to something invalid and it will silently not include any reference info.  This will get caught as part of the attestation in MAA if the customer is expecting UVM info.  Also, tomorrow we're expecting a production signed UVM reference info, so it will all start working.